app/views/users/new.html.haml

```diff
@@ -1,15 +1,22 @@
 %section.register.container
   .row
-    .col-sm-10.col-sm-offset-1
-      = bootstrap_form_for @user,
-                           layout: :horizontal,
-                           label_col: 'col-sm-2',
-                           control_col: 'col-sm-10' do |f|
-        %header
-          %h1 Register
-        %fieldset
-          = f.email_field :email, label: "Email Address"
-          = f.password_field :password
-          = f.text_field :full_name,label: "Full Name"
-          = f.form_group do
-            = f.submit "Sign Up"
+    = form_for @user,layout: :horizontal do |f|
+      %header
+        %h1 Register
+      %fieldset
+        - if @token
+          = f.hidden_field :token, value: @token
+        .form-group.row
+          = f.label :email, "Email Address", class: 'col-form-label col-sm-2'
```

Comment:
Nope, no odd alignments. Though, now that I look at it, the alignment on the log in page is slightly off, I'll look into that.